.pre-commit-config.yaml

```diff
@@ -0,0 +1,13 @@
+default_language_version:
+  python: python3
+repos:
+  - repo: https://github.com/psf/black-pre-commit-mirror
+    rev: 24.8.0
+    hooks:
+      - id: black
```

Comment:
**Question**: Since we're using _ruff_ for linting, how do you feel about using it for formatting, too? The output is mostly the same as with _black_.

Reply:
Sounds like a good idea - done ✅